init/eessi_archdetect.sh

```diff
@@ -138,6 +145,6 @@ shift "$(($OPTIND -1))"
 ARGUMENT=${1:-none}
 
 case "$ARGUMENT" in
-    "cpupath") cpupath; exit;;
-    *) echo "$USAGE"; log "ERROR" "Missing <action> argument";;
+    "cpupath") cpupath $all; exit;;
```

Comment:
@ocaisa Rather than passing an argument to `cpupath`, I would make `-a` change the default value of a `$CPUPATCH_RESULT` from `best` to `all`, which matches better with what's done with `$LOG_LEVEL`?

Reply:
Should be addressed in https://github.com/EESSI/software-layer/pull/264/commits/7db0616a0c78f40864507d77085817af6eb7028a